static/app/views/issueList/savedIssueSearches.tsx

```diff
@@ -174,13 +224,20 @@ const StyledSidebar = styled('aside')`
   }
 `;
 
-const Heading = styled('h2')`
+const HeadingContainer = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 38px;
```

Comment:
do we have to specify a height?

Reply:
am specifying a height here to make sure each heading is the same height - the saved searches header has the `+` button which makes it slightly larger and thus further away from the content below

Reply:
got you